mautic/mautic
# New lead field condition in campaign - Notifications

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | n
| New feature? | y
| Related user documentation PR URL | n
| Related developer documentation PR URL | n
| Issues addressed (#s or URLs) | n
| BC breaks? | n
| Deprecations? | n

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Added support for notification lead field condition in campaign.

![notifications](https://cloud.githubusercontent.com/assets/462477/21880668/7c0846f6-d8a0-11e6-999d-d15e01daa978.PNG)


#### Steps to test this PR:
1. Apply PR
2.  Run mautic:assets:generate
3. See new lead field condition in campaign and test it
